### PR TITLE
Introduce round timeout increase in docs.

### PR DESCRIPTION
**Pull request by dvush**
https://github.com/exonum/exonum/pull/848

### CODE REVIEW

File: src/architecture/consensus.md [hunk 1]
```diff
@@ -59,9 +59,9 @@ The process of reaching consensus on the next block (at the blockchain height
 `H`) consists of several **rounds**, numbered from 1\. The first round starts
 once the
 validator commits the block at height `H - 1`. The onsets of rounds are
-determined
-by a fixed timetable: rounds start after regular intervals. As there is no
-global time, rounds may start at different time for different validators.
+determined by the following timetable: rounds start after linearly increasing
+with round number time intervals. As there is no global time,
```
Comment (slowli) [inline src/architecture/consensus.md]: Nit: For clarity, I'd rephrase as

> rounds start with time intervals that linearly increase with the round number.

File: src/advanced/consensus/specification.md [hunk 2]
```diff
@@ -351,7 +351,10 @@ round are placed into a separate queue (`queued`).
 
 - If the timeout does not match the current height and round, skip further
   timeout processing.
-- Add a timeout (its length is specified by `round_timeout`) for the next round.
+- Add a timeout (its length is specified by the following formula
+  `first_round_timeout + (N-1)*round_timeout_increase`) for the Nth round.
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: For additional clarity, I'd suggest to reorganize the formula as

```none
first_round_timeout * (1 + (N - 1) * q))
```

where `q = 0.1`, if I understand correctly.

File: src/advanced/consensus/specification.md [hunk 3]
```diff
@@ -74,8 +74,8 @@ message processing.
 - `propose_timeout`  
   Proposal timeout after a new block is committed to the blockchain locally.
 
-- `round_timeout`  
-  Interval between algorithm rounds.
+- `first_round_timeout`  
+  Initial interval between algorithm rounds at specific height.
```
Comment (slowli) [inline src/advanced/consensus/specification.md]: Nit: "at specific height" looks slightly confusing; maybe, it could make sense to use "...at a new blockchain height" or simply omit it?

Comment (dvush) [reply src/advanced/consensus/specification.md]: I think it should be mentioned that round timeouts start from initial value after block commit. But I'll try to rephrase it as you suggested.

File: src/architecture/consensus.md [hunk 4]
```diff
@@ -59,9 +59,10 @@ The process of reaching consensus on the next block (at the blockchain height
 `H`) consists of several **rounds**, numbered from 1\. The first round starts
 once the
 validator commits the block at height `H - 1`. The onsets of rounds are
-determined
-by a fixed timetable: rounds start after regular intervals. As there is no
-global time, rounds may start at different time for different validators.
+determined by the following timetable: rounds start with time intervals
+that linearly increase with the round number.
+As there is no global time,
+rounds may start at different time for different validators.
```
Comment (ilammy) [inline src/architecture/consensus.md]: Props for using semantic line breaking. I'd like to see that more often with text files.
